### PR TITLE
Validate at least single channel is provided, remove references to wildfly fpls

### PR DESCRIPTION
**Pull request by TomasHofman**
Cc @spyrkob

### CODE REVIEW

File: prospero-common/src/main/java/org/wildfly/prospero/api/ProvisioningDefinition.java [hunk 1]
```diff
@@ -69,7 +70,7 @@ private ProvisioningDefinition(Builder builder) throws ArtifactResolutionExcepti
                 this.includedPackages.addAll(featurePackInfo.packages);
                 this.repositories.addAll(featurePackInfo.repositories);
                 setUpBuildEnv(channelRepo, provisionConfigFile, channel, featurePackInfo.channelGav);
-            } else {
+            } else if (provisionConfigFile.isPresent()) {
```
Comment (TomasHofman) [inline prospero-common/src/main/java/org/wildfly/prospero/api/ProvisioningDefinition.java:73]: This is just resolving unchecked `provisionConfigFile.get()` on line 76 bellow, which could produce a NPE.

File: prospero-common/src/main/java/org/wildfly/prospero/api/WellKnownFeaturePacks.java [hunk 2]
```diff
@@ -13,11 +14,6 @@
  */
 public enum WellKnownFeaturePacks {
 
-    WILDFLY("wildfly@maven(org.jboss.universe:community-universe):current",
```
Comment (spyrkob) [inline prospero-common/src/main/java/org/wildfly/prospero/api/WellKnownFeaturePacks.java:16]: Can you keep WILDFLY in for now?

Comment (TomasHofman) [reply prospero-common/src/main/java/org/wildfly/prospero/api/WellKnownFeaturePacks.java:16]: Yes, sure.

Comment (TomasHofman) [reply prospero-common/src/main/java/org/wildfly/prospero/api/WellKnownFeaturePacks.java]: Updated.

**Review by spyrkob — APPROVED** [review]

**Pull request merged**